src/tests/ftest/control/config_generate_output.py

```diff
@@ -448,7 +455,8 @@ def test_net_class(self):
         for num_engines in range(1, ib_count + 1):
             # dmg config generate should pass.
             result = dmg.config_generate(
-                access_points="wolf-a", num_engines=num_engines, net_class="infiniband")
+                access_points="wolf-a", num_engines=num_engines, net_class="infiniband",
+                    net_provider=self.def_provider)
```

Comment:
flake wants these to be aligned
```suggestion
                access_points="wolf-a", num_engines=num_engines, net_class="infiniband",
                net_provider=self.def_provider)
```